fairscale/experimental/wgit/signal_sparsity.py

```diff
@@ -52,28 +108,81 @@ class SignalSparsity:
             conv.weight.data = 3d_sparser.get_sst_dst_weight(conv.weight.data)
     """
 
-    def __init__(self) -> None:
-        pass
+    def __init__(
+        self,
+        algo: Algo = Algo.FFT,
+        sst_top_k_dim: Optional[int] = -1,
+        sst_top_k_element: Optional[int] = None,
+        sst_top_k_percent: Optional[float] = None,
```

Comment:
having both none is a good default?

Reply:
good common case is important.

Reply:
Could you make a comment on what you think about this? 
https://github.com/facebookresearch/fairscale/pull/1034#discussion_r927140788

Reply:
sorry, just did. I didn't see your comment there first. I think your idea is good.